### PR TITLE
manifest.json

### PR DESCRIPTION
**Pull request by Im7S1**
add discord id
user stats

## Type of change

- [ ] Add new directory for a server
- [ stats player - discord id ] Update directory of a server
- [ teamspeak ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [ ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/blocksmc/manifest.json [hunk 1]
```diff
@@ -12,10 +12,14 @@
     "tiktok": "blocksmctiktok",
     "instagram": "BlocksMC",
     "discord": "https://discord.gg/BlocksMC",
```
Comment (maaaathis) [inline minecraft_servers/blocksmc/manifest.json:14]:
```suggestion
    "discord": "https://discord.gg/BlocksMC"
```

File: minecraft_servers/blocksmc/manifest.json [hunk 2]
```diff
@@ -12,10 +12,14 @@
...
-    "teamspeak": "ts.blocks.host"
   },
   "brand": {
      "primary": "#001414",
      "background": "#008cd3"
   }
```
Comment (maaaathis) [inline minecraft_servers/blocksmc/manifest.json:19]:
```suggestion
  },
```

**Review by maaaathis — CHANGES_REQUESTED** [review]
And please fix your indentations

**Comment by maaaathis** [comment]
Duplicate of #1237

**Comment by DoJapHD** [comment]
Duplicate

**Pull request closed without merging**